scripts/happy

```diff
@@ -1076,12 +1111,15 @@ def list_command(ctx):
     env = ctx.obj["config"].env
     stackmgr = ctx.obj["stack_mgr"]
     print(f"Listing stacks in environment '{env}'")
-    headings = ["Name", "Owner", "Tag", "Status", "URLs"]
+    headings = ["Name", "Owner", "Tags", "Status", "URLs"]
     tp = TablePrinter(headings)
     for name, info in stackmgr.stacks.items():
         url = info.outputs.get("frontend_url", "")
         status = info.status
-        tp.add_row([name, info.meta.owner, info.meta.imagetag, status, url])
+        tag = info.meta.imagetag
+        if info.meta.imagetags:
+            tag = ", ".join(set(info.meta.imagetags.values()) | {tag})
+        tp.add_row([name, info.meta.owner, tag, status, url])
```

Comment:
Include the default and overridden tag names in the output we display for `happy list`